plugins/woocommerce/client/admin/client/marketplace/components/footer/footer.tsx

```diff
@@ -3,38 +3,66 @@
  */
 import { __ } from '@wordpress/i18n';
 import { check, commentContent, shield, people } from '@wordpress/icons';
-import { createInterpolateElement } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
 
 /**
  * Internal dependencies
  */
 import './footer.scss';
 import IconWithText from '../icon-with-text/icon-with-text';
 import { MARKETPLACE_HOST } from '../constants';
+import { TrackedLink } from '../../../components/tracked-link/tracked-link';
 
-export const refundPolicyTitle = createInterpolateElement(
-	__( '30-day <a>money-back guarantee</a>', 'woocommerce' ),
-	{
-		// eslint-disable-next-line jsx-a11y/anchor-has-content
-		a: <a href={ MARKETPLACE_HOST + '/refund-policy/' } />,
-	}
-);
+export const refundPolicyTitle = ( location: string ) => {
+	return (
+		<TrackedLink
+			targetUrl={ MARKETPLACE_HOST + '/refund-policy/' }
+			onClickCallback={ () =>
+				recordEvent( `marketplace_${ location }_link_click`, {
+					feature_clicked: 'money_back_guarantee',
+				} )
+			}
+			message={ __(
+				'30-day {{Link}}money-back guarantee{{/Link}}',
+				'woocommerce'
+			) }
+		/>
+	);
+};
 
-export const supportTitle = createInterpolateElement(
-	__( '<a>Get help</a> when you need it', 'woocommerce' ),
-	{
-		// eslint-disable-next-line jsx-a11y/anchor-has-content
-		a: <a href={ MARKETPLACE_HOST + '/docs/' } />,
-	}
-);
+export const supportTitle = ( location: string ) => {
+	return (
+		<TrackedLink
+			targetUrl={ MARKETPLACE_HOST + '/docs/' }
+			onClickCallback={ () =>
+				recordEvent( `marketplace_${ location }_link_click`, {
+					feature_clikced: 'get_help',
```

Comment:
```suggestion
					feature_clicked: 'get_help',
```

Also fixed if you decide to apply my patch.